api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -114,15 +117,17 @@
 
     @Autowired
     CohortReviewController(CohortReviewService cohortReviewService,
-                           BigQueryService bigQueryService,
-                           CodeDomainLookupService codeDomainLookupService,
-                           ParticipantCounter participantCounter,
-                           Provider<GenderRaceEthnicityConcept> genderRaceEthnicityConceptProvider) {
+        BigQueryService bigQueryService,
+        CodeDomainLookupService codeDomainLookupService,
+        ParticipantCounter participantCounter,
+        Provider<GenderRaceEthnicityConcept> genderRaceEthnicityConceptProvider,
+        CdrVersionContext cdrVersionContext) {
         this.cohortReviewService = cohortReviewService;
         this.bigQueryService = bigQueryService;
         this.codeDomainLookupService = codeDomainLookupService;
         this.participantCounter = participantCounter;
         this.genderRaceEthnicityConceptProvider = genderRaceEthnicityConceptProvider;
+        this.cdrVersionContext = cdrVersionContext;
```

Comment:
Do we need to inject the cdrVersionContext? Since all methods are static we shouldn't have to inject a reference.

Reply:
Removed.